### ISSUE TITLE
"Auto pair common Markdown syntax" not working for dead backtick (`` ` ``) key on Ubuntu

### ISSUE THREAD

**Issue by LinqLover** [typora/typora-issues#6027]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
On Ubuntu with a German keyboard layout (Qwertz), I cannot type a grave accent (`` ` ``) while selecting some text to wrap it into a code block.

**To Reproduce**
1. Type `abc`
2. Select it
3. Type dead backtick on Ubuntu Qwertz keyboard:
   - either <kbd>Shift</kbd> + <kbd>´</kbd> , <kbd>Shift</kbd> + <kbd>´</kbd>
   - or <kbd>Shift</kbd> + <kbd>´</kbd> , <kbd>Space</kbd>

**Expected behavior**
`` `abc` ``

**Actual behavior**
`` ` ``

**Desktop (please complete the following information):**
 - OS: Ubuntu 22, Qwertz keyboard layout

**Typora Version**
e.g: 1.8.10

**Additional context**
For reference, VS Code handles both input versions exactly as I would expect it.

Not sure what's the proper way to implement this, but from my black-box perspective I think U+0300 COMBINING GRAVIS ACCENT character should be handled in a similar way to U+0060, but this might need to be honored for the next keyboard input. Also consider special cases like pressing <kbd>Shift</kbd> + <kbd>´</kbd> , <kbd>a</kbd> while the text is selected, which like in VS Code should replace the prior selection by a single `à`.

**Comment by abnerlee** [comment]
This option requires a restart of Typora to apply. Does this happens in new Typora window or after you restart Typora?

**Comment by LinqLover** [comment]
Yes, a restart of Typora does not fix this. This is really specific to the dead key behavior. I'm sorry for the late reply.
